wsd/ServerAuditUtil.cpp

```diff
@@ -0,0 +1,32 @@
+/* -*- Mode: C++; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4; fill-column: 100 -*- */
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include "ServerAuditUtil.hpp"
+
+std::list<std::pair<std::string, std::string>> ServerAuditUtil::entries;
+
+std::string ServerAuditUtil::getResultsJSON()
+{
+    std::string result = "{\"serverAudit\": [";
+
+    for (auto entry : entries)
+        result += "{\"code\": \"" + entry.first + "\", \"status\": \"" + entry.second + "\"}";
+
+    result += "]}";
+    return result;
+}
+
+void ServerAuditUtil::addEntry(std::string code, std::string status)
+{
+    entries.push_back(std::make_pair(code, status));
```

Comment:
Is this thread safe?

Reply:
moved static map to DocumentBroker now, so it is not shared